src/wallet/wallet.cpp

```diff
@@ -2385,10 +2385,10 @@ CAmount CWallet::GetNormalizedAnonymizedBalance() const
 
     LOCK2(cs_main, cs_wallet);
     for (const auto& outpoint : setWalletUTXO) {
-        std::map<uint256, CWalletTx>::const_iterator it = mapWallet.find(outpoint.hash);
-        if (it == mapWallet.end()) continue;
-        if (!IsDenominated(outpoint)) continue;
-        if (it->second.GetDepthInMainChain() < 0) continue;
+        const auto it = mapWallet.find(outpoint.hash);
```

Comment:
fyi, `const` is implicitly part of the type if you just use `auto`. This is due to the method being `const` and accessing a member here. Or do you want to make this explicitly visible for readers of the code? (IMHO not needed, as it's usually not important what's const when reading code...it only becomes important when modifying/writing code).

Seen this very often in many other places as well (e.g. the `const auto& outpoint` one line above)

Reply:
I agree that it's probably an overkill but still prefer this as an additional safety check tbh.